modules/saved_searches/setup.php

```diff
@@ -10,7 +10,7 @@
 add_handler('ajax_save_search', 'save_search', true, 'core');
 add_handler('ajax_save_search', 'language', true);
 add_handler('ajax_save_search', 'date', true, 'core');
-add_handler('ajax_save_search', 'http_headers', true, 'core');
```

Comment:
This looks like an inadvertent change ... ?

Reply:
exactly, I made few changes and fix that mistake